spec/features/stories_manage_spec.rb

```diff
@@ -39,7 +39,7 @@
     expect(page).to have_content "Story updated!"
   end
 
-  it "allows me to delete a story", js: true do
+  it "allows me to delete a story" do
```

Comment:
We don't want to test these with Javascript disabled as in `estimates_manage_spec.rb` ?

Reply:
I tried to write a test to delete a story without JavaScript enabled, but it doesn't look like deleting stories works without JavaScript - when I try to delete a story in my browser with JS disabled, I get the following error: 
<img width="1241" alt="Screen Shot 2022-01-18 at 1 39 53 PM" src="https://user-images.githubusercontent.com/6892410/149998734-88234445-c01f-4871-a52b-f95bdb92b4e8.png">